formatting.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Run it before committing to verify whether all files 
+# are properly formatted so you won't fail early on review
+# Execute from repo root or, if using aa from base/head/startup-os
+
+RED=$(tput setaf 1)
+RESET=$(tput sgr0)
+
+npm install &>/dev/null
+
+bazel run //tools/formatter -- \
+					--path $(pwd) \
+					--java --python --proto --cpp --build \
+					--ignore_directories $(find $(pwd) -name node_modules -type d | paste -s -d , -) \
+					&>/dev/null
+if [[ ! -z $(git status -s) ]]; then
+	echo "$RED[!] Source files are not formatted$RESET";
+	echo "Please run ''./formatting.sh'' to fix it"
```

Comment:
You merged `check` and `fix`?
So if it's called `formatting.sh` now, why would I run it again if I just ran it? (`Please run ''./formatting.sh'' to fix it`)

Reply:
@oferb I think it'd be reasonable to either take `check`|`fix` as cmdline argument or to always fix and print out error if we're on CI

Reply:
Let's always fix then, but maybe add in a comment how to make it just check.